### PR TITLE
Reduce launch time by changing pods to static libraries

### PR DESCRIPTION
**Pull request by alloy**
Switches us to CocoaPods master to use https://github.com/CocoaPods/CocoaPods/pull/6966.

Shaves 1.5 sec off of launch time.

Yay.

😝

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          3 Errors
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Developer Specific file shouldn’t be changed</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Release 4.0.3 does not have a date</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">You need to set the App’s plist version to 4.0.4 - use <code>make update_bundle_version</code></td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Needs testing on a Phone if change is non-trivial</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_travis">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: Artsy.xcodeproj/xcshareddata/xcschemes/Artsy.xcscheme [hunk 1]
```diff
@@ -130,6 +130,11 @@
             value = "1"
             isEnabled = "NO">
          </EnvironmentVariable>
+         <EnvironmentVariable
+            key = "DYLD_PRINT_STATISTICS"
+            value = "1"
+            isEnabled = "NO">
+         </EnvironmentVariable>
```
Comment (alloy) [inline Artsy.xcodeproj/xcshareddata/xcschemes/Artsy.xcscheme]: This is what the “Developer Specific file” Danger error is about. It’s not enabled, but good to have in there, so ¯\_(ツ)_/¯

Comment (orta) [reply Artsy.xcodeproj/xcshareddata/xcschemes/Artsy.xcscheme]: That's fine, if you're alright with it 👍

**Comment by orta** [comment]
Looks like you _might_ need to do Nimble changes 

> /Users/distiller/eigen/Pods/Nimble/Sources/NimbleObjectiveC/NMBStringify.m:2:9: 'Nimble/Nimble-Swift.h' file not found

**Comment by alloy** [comment]
@orta Alright, this is good to go.

File: Artsy/Views/Styled_Subclasses/StyledSubclasses.h [hunk 2]
```diff
@@ -1,4 +1,4 @@
-@import Artsy_UILabels;
-#import <Artsy_UIButtons/ARButtonSubclasses.h>
+#import <Artsy+UILabels/Artsy+UILabels.h>
+#import <Artsy-UIButtons/ARButtonSubclasses.h>
```
Comment (orta) [inline Artsy/Views/Styled_Subclasses/StyledSubclasses.h]: lol yeah, the old `+` vs `-`

File: Podfile [hunk 3]
```diff
@@ -9,7 +9,7 @@ source 'https://github.com/artsy/Specs.git'
 source 'https://github.com/CocoaPods/Specs.git'
 
 platform :ios, '8.0'
-use_frameworks!
+use_frameworks!(false)
```
Comment (orta) [inline Podfile]: is this just what it is without `use_frameworks!`?

**Comment by orta** [comment]
OK, just gonna give this a run locally

**Comment by orta** [comment]
Might need that pod plugin, editing the Pods dir fails on my `pod install` because they're marked as read only, will update this PR.

```sh
Generating Pods project
[!] An error occurred while processing the post-install hook of the Podfile.

Permission denied @ rb_sysopen - Pods/Nimble/Sources/NimbleObjectiveC/CurrentTestCaseTracker.h

/Users/orta/dev/projects/artsy/ios/eigen/Podfile:221:in `write'
/Users/orta/dev/projects/artsy/ios/eigen/Podfile:221:in `block (3 levels) in from_ruby'
/Users/orta/dev/projects/artsy/ios/eigen/Podfile:218:in `each'
/Users/orta/dev/projects/artsy/ios/eigen/Podfile:218:in `block (2 levels) in from_ruby'
```

**Comment by alloy** [comment]
Ah yes, you need to nuke those dirs (or just `Pods`) whenever we patch from `post_install` because the files get locked once installed. The plugin wouldn’t really change that.

**Comment by orta** [comment]
A plugin could check for the locked status and warn that you need to do a full nuke

**Comment by orta** [comment]
Tested on a device, works fine, and is definitely a perceptually faster launch  - nice one

**Comment by alloy** [comment]
True dat. I definitely think a ‘patch’ plugin would be a nice thing to have.

File: Artsy/Views/Styled_Subclasses/StyledSubclasses.h (hunk 2 above)
Comment (segiddins) [reply Artsy/Views/Styled_Subclasses/StyledSubclasses.h]: Please file an issue about this, this change shouldn’t be necessary

File: Podfile (hunk 3 above)
Comment (alloy) [reply Podfile]: Oh good point, I assume so yeah.

**Pull request merged**